src/alpine/3.9/amd64/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM alpine:3.9
+FROM python:3.7-alpine3.9
```

Comment:
I'm not seeing this listed as being a supported tag any longer.  Because of that, I don't think we should be taking a dependency on it.  The consequence of this is the image would not be updated as patches are made to the alpine:3.9 image.

Reply:
Good to know, thanks. Updated.